auth.js

```diff
@@ -0,0 +1,40 @@
+const jwtSecret = 'your_jwt_secret';
+// This has to be the same key used in the JWTStrategy
+const jwt = require('jsonwebtoken'),
+  passport = require('passport');
+
+require('./passport');
+// Your local passport file
+
+let generateJWTToken = (user) => {
+  return jwt.sign(user, jwtSecret, {
+    subject: user.Username,
+    // This is the username you’re encoding in the JWT
+    expiresIn: '7d',
+    // This specifies that the token will expire in 7 days
+    algorithm: 'HS256'
+    // This is the algorithm used to “sign” or encode the values of the JWT
+  });
+}
+
+module.exports = (router) => {
+  router.use(passport.initialize());
+  //Important! Without using this line of code it will throw an error when trying to login with Postman due to updated passport dependency:Changelog Passport 0.5.0 -2021-09-23
+  router.post('/login', (req, res) => {
+    passport.authenticate('local', { session: false }, (error, user, info) => {
+      if (error || !user) {
+        return res.status(400).json({
```

Comment:
If something on login is not right, you probably want to throw a 401 unauthorized